channels/kr.m3u

```diff
@@ -167,3 +167,5 @@ http://54.180.143.6:9981/stream/channelid/1805607194
 http://54.180.143.6:9981/stream/channelid/965307250
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://imgur.com/2TSc2pJ.png" group-title="",Animax
 http://211.49.191.148:9981/stream/channelid/1158483964
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/WRzx3F9.png" group-title="",Fox Life
+http://211.49.191.148:9981/stream/channelid/1960414728
```

Comment:
![afbeelding](https://user-images.githubusercontent.com/25525836/66758023-bd04df00-ee9d-11e9-9b62-e6053f68475e.png)
Link does not work

Reply:
<img width="837" alt="Screen Shot 2019-10-15 at 1 45 43 AM" src="https://user-images.githubusercontent.com/1114030/66785179-89ea3c00-eeed-11e9-857c-0f81969b9ad7.png">
link is working

Reply:
same thing, stream works but it is not possible to watch due to lags